packages/ui-extensions-sdk/src/events/events.ts

```diff
@@ -49,6 +50,13 @@ interface DDEventDataTypes<AuthStateArgs> {
     [EventType.API_ACCESS_CHANGE]: APIAccessChangeEvent;
 }
 
+const deprecationWarnings: Partial<Record<EventType, string>> = {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Unrelated to this change, but we should enable [noUncheckedIndexedAccess](https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess) at some point.